packages/nodejs/src/instrumentation/http/lifecycle/__tests__/outgoing.test.ts

```diff
@@ -0,0 +1,89 @@
+import { TestRegistry } from "../../../../../test/registry"
+import nock from "nock"
+import { HashMap } from "@appsignal/types"
+import {
+  Span,
+  SpanOptions,
+  SpanContext,
+  SpanData
+} from "../../../../interfaces"
+import { BaseClient as Client } from "../../../../client"
+import { BaseTracer as Tracer } from "../../../../tracer"
+import { PLUGIN_NAME, instrument } from "../../http"
+import { RootSpan, ChildSpan } from "../../../../span"
+
+describe("HTTP outgoing requests", () => {
+  const name = "TEST APP"
+  const pushApiKey = "TEST_API_KEY"
+
+  let http: any
+  let client: Client
+  let tracer: Tracer
+
+  // enableMinutelyProbes is set to false so we don't leak timers
+  const DEFAULT_OPTS = {
+    active: true,
+    name,
+    pushApiKey,
+    enableMinutelyProbes: false
+  }
+
+  beforeEach(() => {
+    TestRegistry.clear()
+    client = new Client({ ...DEFAULT_OPTS })
+    tracer = new Tracer()
+    http = require("http")
+    instrument(http, tracer).install()
+  })
+
+  async function performRequest() {
+    return new Promise<HashMap<any>>((resolve, reject) => {
+      const options = {
+        host: "example.com",
+        path: "/foo",
+        method: "GET"
+      }
+      const request = http.request(options, (stream: any) => {
+        const responseStatus = stream.statusCode
+        stream.setEncoding("utf8")
+        let responseBody = ""
+
+        stream
+          .on("data", (chunk: any) => {
+            responseBody += chunk
+          })
+          .on("end", () => {
+            const value = { status: responseStatus, body: responseBody }
+            if (responseStatus !== 200) {
+              reject(value)
+            } else {
+              resolve(value)
+            }
+          })
+      })
+      request.on("error", (error: Error) => {
+        reject({ error: error })
+      })
```

Comment:
If we don't care about the request, other than that we wait for it and it doesn't error, then:

```suggestion
        stream
          .on("data", () => {})
          .on("end", resolve)
      })
      
      request.on("error", reject)
```